README.md

```diff
@@ -0,0 +1,16 @@
+# To run
+
+1. Spin up nodes
+```
+> docker-compose up
```

Comment:
Need to update this, like:
`cd plug-latest`
To show you have to go to a particular docker-compose folder